pkg/apis/v2v/v1alpha1/vmimportconfig_types.go

```diff
@@ -25,6 +25,33 @@ import (
 // VMImportConfigSpec defines the desired state of VMImportConfig
 type VMImportConfigSpec struct {
 	ImagePullPolicy corev1.PullPolicy `json:"imagePullPolicy,omitempty"`
+
+	// Rules on which nodes controller pod(s) will be scheduled
+	Infra NodePlacement `json:"infra,omitempty"`
```

Comment:
Should we touch v1alpha any more? We have newer v1beta.

Reply:
This is what I saw in CDI so I did the same. In the past we had to fix `v1alpha` to have successful HCO deployment.

Reply:
OK